src/chrome/content/rules/atlas.sk.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	For other Centrum.cz coverage, see centrum.cz.xml.
+
+	Nonfunctional subdomains:
+		- (www.)atlas.sk
+-->
+<ruleset name="Atlas.sk">
+	<target host="mail.atlas.sk" />
+	<target host="reg.atlas.sk" />
+	<target host="user.atlas.sk" />
+	<target host="heslo.atlas.sk" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```

Comment:
I have some more to document:
mapy.atlas.sk
automix.atlas.sk
aktualne.atlas.sk
referaty.atlas.sk
wanda.atlas.sk
magazin.atlas.sk
log.atlas.sk
katalog.atlas.sk
cp.atlas.sk
zena.atlas.sk
katalog.menu.atlas.sk
kontakt.atlas.sk
menu.atlas.sk
mapa.menu.atlas.sk
ads.atlas.sk
pohladnice.atlas.sk
slovniky.atlas.sk
blogy.atlas.sk